### PR TITLE
Morse code converter added Fix #24

### LINKED ISSUE #24: Add more encoders/decoders - Morse code
We need more encoders/decoders, ie String to Hex, String to MD5 etc. 

💡 Here are some ideas https://www.browserling.com/tools/

To add an encoder or decoder, follow the [Contributing Guide](https://github.com/nadchif/adobe-brackets-encode-decode/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by okandavut**
Fix #24

### CODE REVIEW

**Comment by okandavut** [comment]
I will check for this issues i could not run to script build i will try on macos.

**Comment by Shankhanil** [comment]
Great work. That was really quick of you. There are some failed checks. You can fix them by using 
`eslint --fix ./src/*` on the project folder and committing the changes again.

**Comment by okandavut** [comment]
> Great work. That was really quick of you. There are some failed checks. You can fix them by using
> `eslint --fix ./src/*` on the project folder and committing the changes again.

I really like this project and i could not run to script on windows but i will try on macos now and i will solve :)

**Comment by okandavut** [comment]
@nadchif I added control for unsupported characters for encode/decode operations. Can you check it again?

**Comment by nadchif** [comment]
Thank you for the prompt response. I will check it out and get back to you soon ☺

**Comment by Shankhanil** [comment]
> @nadchif I added control for unsupported characters for encode/decode operations. Can you check it again?

@okandavut We will soon add a series of unit tests for all the available encoders-decoders. Thank you so much for your contribution

File: src/convertors/morsecode.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    let code = '';
+    for (let i = 0; i < txt.length; i++) {
+      if (symbols[letters.indexOf(txt[i])] != undefined) {
+        code += symbols[letters.indexOf(txt[i])] + ' ';
+        continue;
+      }
+      undefinedCount++;
+    }
+    if (undefinedCount > 0 ) {
+      Dialogs.showModalDialog('',
```
Comment (nadchif) [inline src/convertors/morsecode.js:33]: I've added dialog support, so the user can get notified immediately

Comment (okandavut) [reply src/convertors/morsecode.js:33]: Yes thanks! :)

**Comment by nadchif** [comment]
Thanks again! @all-contributors please add @okandavut for code

**Comment by nadchif** [comment]
@all-contributors please add @okandavut for code

**Pull request merged**